otel/otel-configuration-of-instrumentation.adoc

```diff
@@ -1,8 +1,8 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="otel-instrumentation"]
-= Configuring and deploying the OpenTelemetry instrumentation injection
+[id="otel-configuration-of-instrumentation"]
+= Configuration of instrumentation for the {OTELShortName}
```

Comment:
```suggestion
= Configuring instrumentation
```
Again, suggest shorter title with gerund

Reply:
This is my personal preference too, but after past feedback from other reviewers, I only use gerunds for procedure content.

Reply:
See previous comment